.devcontainer/Dockerfile

```
@@ -0,0 +1,23 @@
+FROM node:20
+
+ARG TZ
+ENV TZ="$TZ"
+
+RUN apt update && apt install -y less git procps sudo fzf zsh man-db unzip gnupg2 gh iptables ipset iproute2 dnsutils aggregate jq rsync
+
+RUN mkdir -p /usr/local/share/npm-global && chown -R node:node /usr/local/share
+ARG USERNAME=node
+RUN SNIPPET="export PROMPT_COMMAND='history -a' && export HISTFILE=/commandhistory/.bash_history" && mkdir /commandhistory && touch /commandhistory/.bash_history && chown -R $USERNAME /commandhistory
```

Hadolint (2.12.0)

[warning] SNIPPET appears unused. Verify use (or export if used externally).

(SC2034)